### ISSUE TITLE
Last line of output should end in newline

### ISSUE THREAD

**Issue by pesterhazy** [bhb/expound#8]
Used from the terminal:

```
boot.user=> (expound/expound :example/place {})
-- Spec failed --------------------

  {}

should contain keys: `:city`,`:state`

-- Relevant specs -------

:example/place:
  (clojure.spec.alpha/keys
   :req-un
   [:example.place/city :example.place/state])

-------------------------
Detected 1 errornil
```

The output (`nil`) is glued to the text. I think the last line printed by `expound` should end with a newline (like `s/explain`).

(To reproduce I use `BOOT_VERSION=2.7.2-SNAPSHOT BOOT_CLOJURE_VERSION=1.9.0-alpha17 rlwrap boot -d expound bare-repl`)

**Comment by bhb** [comment]
@pesterhazy Thanks for reporting this! I can reproduce this in boot and I'll look into it shortly.

**Comment by bhb** [comment]
@pesterhazy Thanks again for reporting this! It's been fixed in master. If this is blocking you, I can release a new version, otherwise I will add a few more features and release next week.

**Comment by pesterhazy** [comment]
Not blocked :) Thanks for the fix! Nice tests btw

**Comment by bhb** [comment]
I have released version 0.2.0 which includes this fix. Thanks for the help! https://clojars.org/expound/versions/0.2.0
